43761.2 <c-42#236> Created logger
43761.2 <c-42#236:7> Logger was updated
43761.2 <c-42#236:7> Started intializing cube lib
43761.2 <c-42#236:7> Finished intializing cube lib with worker
43761.2 <c-42#236:7> shareCubes is called
43761.2 <c-42#236:7> Added 0 failed local cubes
43761.2 <c-42#236:7> Pruned 0 local cubes
43761.2 <c-42#236:7> CubeSolverThread entered cube retrieval loop, currently there are 0 cubes
43761.2 <c-42#236:7> returnFailedAndRequestCubes => [38] {}
43761.2 <c-42#236:7> Sent 0 failed cubes to root and requested new cubes
43761.2 <c-42#236:7> Received 4 cubes from root
43761.2 <c-42#236:7> CubeSolverThread entered cube retrieval loop, currently there are 4 cubes
43761.2 <c-42#236:7> CubeSolverThread retrieved 4 cubes
43761.2 <c-42#236:7> Started solving a cube
43774.1 <c-42#236:7> Cube failed
43774.1 <c-42#236:7> Added failed cube
43774.1 <c-42#236:7> Started solving a cube
43785 <c-42#236:7> Cube failed
43785 <c-42#236:7> Added failed cube
43785 <c-42#236:7> Started solving a cube
43797.5 <c-42#236:7> Cube failed
43797.5 <c-42#236:7> Added failed cube
43797.5 <c-42#236:7> Started solving a cube
43802.9 <c-42#236:7> appl_interrupt was called
43802.9 <c-42#236:7> appl_interrupt was called
43802.9 <c-42#236:7> appl_withdraw was called
43802.9 <c-42#236:7> Started withdraw thread
43802.9 <c-42#236:7> STOP (0.00s since last cb)
43802.9 <c-42#236:7> Solving interrupted
43802.9 <c-42#236:7> Leaving the main loop
43802.9 <c-42#236:7> Enter destructor of CubeWorkerInterface
43802.9 <c-42#236:7> Enter destructor of CubeRoot
43802.9 <c-42#236:7> Finished withdraw thread
43864.8 <c-42#236:7> Enter destructor
43864.8 <c-42#236:7> Time working: 41.637
43864.8 <c-42#236:7> Time suspended: 0.000
43864.8 <c-42#236:7> Joined cleanup thread
43864.8 <c-42#236:7> Exit destructor
43864.8 <c-42#236:7> Destructing logger
